frontend/src/index.css

```
@@ -6557,27 +6868,27 @@ textarea {
 .requests-page__review-path,
 .requests-page__review-reason {
   min-width: 0;
-  color: var(--aurral-gray);
+  color: var(--aurral-text-subtle);
   font-size: 0.6875rem;
   line-height: 1.35;
   overflow-wrap: anywhere;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)